proctor-common/src/main/java/com/indeed/proctor/common/ProctorResult.java

```diff
@@ -193,4 +197,8 @@ public Identifiers getIdentifiers() {
     public Map<String, Object> getInputContext() {
         return inputContext;
     }
+
+    public boolean markTestAsLogged(String test) {
```

Comment:
need a getter method also since set is private

Reply:
We don't have a usecase for getter method yet

Reply:
Before emitting log won't you want to check if the set contains the testVersion? 

Guess getter not actual need but a contains method

Reply:
Just call markTestAsLogged(test) will serve the purpose as it returns a boolean

Reply:
Missed, resolved.